kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Message.kt

```diff
@@ -183,27 +269,23 @@ public suspend fun Message.requireChannel(
     deleteOriginal: Boolean = true,
     deleteResponse: Boolean = true
 ): Boolean {
-    val topRole = if (getGuildOrNull() != null) {
-        this.getAuthorAsMember()!!.getTopRole()
-    } else {
+    val topRole = if (getGuildOrNull() == null) {
         null
-    }
+    } else getAuthorAsMember()!!.getTopRole()
```

Comment:
Again, braces please. Also, I'm not sure why we're getting rid of `this` everywhere — I find it often makes things easier to understand, but I don't seem to be able to lint for it.